fpgaperf.py

```diff
@@ -851,9 +859,9 @@ def main():
     parser.add_argument('--device', default='hx8k', help='Device within family')
     parser.add_argument('--package', default=None, help='Device package')
     parser.add_argument('--strategy', default='default', help='Optimization strategy')
-    parser.add_argument('--toolchain', help='Tools to use')
+    parser.add_argument('--toolchain', help='Tools to use', choices=get_toolchains())
     parser.add_argument('--list-toolchains', action='store_true', help='')
-    parser.add_argument('--project', help='Source code to run on')
+    parser.add_argument('--project', help='Source code to run on', choices=get_projects())
     parser.add_argument('--list-projects', action='store_true', help='')
     parser.add_argument('--seed', default=None, help='31 bit seed number to use, possibly directly mapped to PnR tool')
```

Comment:
ideally we should also choices=get_seedable()

Reply:
`--seed` takes an integer? Are you saying check that `--seed` is only used on projects which have `get_seedable()`?

Reply:
er yeah nvm